fe/fe-core/src/main/java/com/starrocks/sql/analyzer/PipeAnalyzer.java

```diff
@@ -164,6 +167,8 @@ public static void analyze(CreatePipeStmt stmt, ConnectContext context) {
         stmt.setInsertSql(insertSql);
         InsertAnalyzer.analyze(insertStmt, context);
 
+        analyzePipeName(stmt.getPipeName(), insertStmt.getTableName().getDb());
```

Comment:
I want to know that, will the `db` of insertStmt be set if the db is not specified?

Reply:
Yes. The insert stmt's database would be set as the current database.